### ISSUE TITLE
Plugin not working on Android

### ISSUE THREAD

**Issue by YukiGasai** [timrogers/obsidian-share-as-gist#25]
Hi I really like the functionality of your plugin and would like to use it on mobile too. 
But when I enable the plugin, I get this error message.

```red
app.js:1 Plugin failure: obsidian-share-as-gist TypeError: Cannot read properties of undefined (reading 'homedir')
    at node_modules/clean-stack/index.js (plugin:obsidian-share-as-gist:9394:29)
    at __require (plugin:obsidian-share-as-gist:33:50)
    at node_modules/aggregate-error/index.js (plugin:obsidian-share-as-gist:9422:22)
    at __require (plugin:obsidian-share-as-gist:33:50)
    at node_modules/@octokit/webhooks/dist-node/index.js (plugin:obsidian-share-as-gist:9525:42)
    at __require (plugin:obsidian-share-as-gist:33:50)
    at node_modules/@octokit/app/dist-node/index.js (plugin:obsidian-share-as-gist:9881:22)
    at __require (plugin:obsidian-share-as-gist:33:50)
    at anonymous (plugin:obsidian-share-as-gist:12099:26)
    at t.<anonymous> (app.js:1:1517602)
```

**Comment by timrogers** [comment]
@YukiGasai Can you try again now? I think I may have fixed it - or at least done something to surface the real problem.

**Comment by YukiGasai** [comment]
Okay, this made some progress. I can activate the plugin now without an error.

There is a new error when running the share public or private commands.

The gist is not being created.

```red
plugin:obsidian-share-as-gist:3297 Uncaught (in promise) ReferenceError: Buffer is not defined
    at exports2.toBuffer (plugin:obsidian-share-as-gist:3297:42)
    at module2.exports (plugin:obsidian-share-as-gist:3469:40)
    at matter3 (plugin:obsidian-share-as-gist:3520:18)
    at getSharedGistsForFile (plugin:obsidian-share-as-gist:6534:51)
    at eval (plugin:obsidian-share-as-gist:6567:31)
    at Generator.next (<anonymous>)
    at eval (plugin:obsidian-share-as-gist:75:61)
    at new Promise (<anonymous>)
    at __async (plugin:obsidian-share-as-gist:59:10)
    at Object.eval [as editorCallback] (plugin:obsidian-share-as-gist:6559:59)
```

**Comment by timrogers** [comment]
@YukiGasai Can you give this another go now? I think I have solved the `Buffer` issue in v1.0.2.
